pkg/kubelet/container/runtime.go

```diff
@@ -309,3 +309,21 @@ func ParsePodFullName(podFullName string) (string, string, error) {
 	}
 	return parts[0], parts[1], nil
 }
+
+// ToAPIPod converts Pod to api.Pod. Note that if a field in api.Pod has no
+// corresponding field in Pod, the field would not be populated.
+func ToAPIPod(pod *Pod) *api.Pod {
```

Comment:
nit: Doesn't it make more sense to have this be a member of Pod?  I don't feel terribly strongly about this one.